src/Table/Table.tsx

```diff
@@ -60,6 +71,12 @@ export function Table<RowShape>({
         width: "100%",
       }}
     >
+      <colgroup>
+        {columns.map(({ colProps }, index) => (
+          <col key={index} {...colProps} />
```

Comment:
Using an `index` for a `key` is an anti-pattern. I didn't even realize our linter allowed this. It won't matter here until we change the columns (like if we offer re-ordering of the columns, which we might do). There's always a better way. In this case, use the `column`'s `id`.